iconservice/builtin_scores/governance/governance.py

```diff
@@ -52,6 +52,7 @@
 
 CONTEXT_TYPE_INVOKE = 'invoke'
 CONTEXT_TYPE_QUERY = 'query'
+ZERO_TX_HASH = bytes(32)
```

Comment:
have to add this code from governance SCORE to https://github.com/icon-project/governance after merge to develop

Reply:
I found a better way to resolve this problem without modifying the existing governance SCORE code.

IconScoreDeployStorage.get_tx_hashes_by_score_address() will convert ZERO_TX_HASH to None instead.
So, we don't need to modify the governance SCORE code anymore.